apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/config/SwiftAuthenticationModule.java

```diff
@@ -98,12 +97,16 @@ protected void configure() {
       @GET
       @Consumes
       @ResponseParser(AdaptTempAuthResponseToAccess.class)
-      Access auth(@HeaderParam(STORAGE_USER) String user, @HeaderParam(STORAGE_PASS) String key);
+      Access auth(@BinderParam(TempAuthBinder.class) Credentials credentials);
    }
 
    static final class AdaptTempAuthResponseToAccess
          implements Function<HttpResponse, Access>, InvocationContext<AdaptTempAuthResponseToAccess> {
 
+      @Inject
+      @Named(TEMP_AUTH_HEADER_USER)
+      private String identityHeaderNameUser;
```

Comment:
Add it to the constructor and declare it final to use constructor injection too.